### PR TITLE
Fix typos in the pg_proc catalog DESC fields

### PR DESCRIPTION
**Pull request by danielgustafsson**
The catullus script had not been used to synchronize all pg_proc changes into the catalog, so ensure we are in sync before we cut Beta and (mostly) freeze the catalog. Also fix a typo in another DESC property which was so far not out of sync.

### CODE REVIEW

File: src/include/catalog/pg_proc_gp.h [hunk 1]
```diff
@@ -807,15 +807,15 @@ DATA(insert OID = 7596 ( complex_gte  PGNSP PGUID 12 1 0 0 0 f f f f t f i 2 0 1
...
 
 /* gp_hyperloglog_out(counter gp_hyperloglog_estimator) => cstring */
 DATA(insert OID = 7159 ( gp_hyperloglog_out  PGNSP PGUID 12 1 0 0 0 f f f f t f i 1 0 2275 "7157" _null_ _null_ "{counter}" _null_ gp_hyperloglog_out _null_ _null_ _null_ n a ));
-DESCR("Encode a hyperloglog counter into a bytea");
+DESCR("Encode an gp_hyperloglog_counter into a bytea");
 
 /* gp_hyperloglog_comp(counter gp_hyperloglog_estimator) => gp_hyperloglog_estimator */
 DATA(insert OID = 7160 ( gp_hyperloglog_comp  PGNSP PGUID 12 1 0 0 0 f f f f t f i 1 0 7157 "7157" _null_ _null_ "{counter}" _null_ gp_hyperloglog_comp _null_ _null_ _null_ n a ));
-DESCR("Compress a hyperloglog counter");
+DESCR("Compress an hyperloglog counter");
```
Comment (tvesely) [inline src/include/catalog/pg_proc_gp.h]: This actually seems to be incorrect...

https://www.a-or-an.com/a_an/hyper

Comment (danielgustafsson) [reply src/include/catalog/pg_proc_gp.h]: "A hyper.." sounds better to my non-native-speaking mind. This seems to have been semi-fixed in 28a271253efb209cb84f1a7195ea2cbeac8d6408 but not all the way.

**Comment by kmacoskey** [comment]
> This looks good. Let's hurry and get it in before we cut beta.

🙁

**Comment by danielgustafsson** [comment]
> > This looks good. Let's hurry and get it in before we cut beta.
> 
> 🙁

If we don't, it's a catalog change and thats not terribly good.

File: src/include/catalog/pg_proc_gp.h (hunk 1 above)
Comment (tvesely) [reply src/include/catalog/pg_proc_gp.h]: My vote is definitely "a hyperloglog". We should definitely pick one and change it on all of them.

File: src/include/catalog/pg_proc_gp.h [hunk 2]
```diff
@@ -807,15 +807,15 @@ DATA(insert OID = 7596 ( complex_gte  PGNSP PGUID 12 1 0 0 0 f f f f t f i 2 0 1
 
 /* gp_hyperloglog_in(value cstring) => gp_hyperloglog_estimator */
 DATA(insert OID = 7158 ( gp_hyperloglog_in  PGNSP PGUID 12 1 0 0 0 f f f f t f i 1 0 7157 "2275" _null_ _null_ "{value}" _null_ gp_hyperloglog_in _null_ _null_ _null_ n a ));
-DESCR("Decode a bytea into a hyperloglog counter");
+DESCR("Decode a bytea into gp_hyperloglog_counter");
 
 /* gp_hyperloglog_out(counter gp_hyperloglog_estimator) => cstring */
 DATA(insert OID = 7159 ( gp_hyperloglog_out  PGNSP PGUID 12 1 0 0 0 f f f f t f i 1 0 2275 "7157" _null_ _null_ "{counter}" _null_ gp_hyperloglog_out _null_ _null_ _null_ n a ));
-DESCR("Encode a hyperloglog counter into a bytea");
+DESCR("Encode an gp_hyperloglog_counter into a bytea");
```
Comment (jchampio) [inline src/include/catalog/pg_proc_gp.h]: Nitpick: `Encode an gp_hyperloglog_counter into a bytea` vs `Decode a bytea into gp_hyperloglog_counter` (note lack of article) seems inconsistent.

Also, should it be "**a** gp_hyperloglog_counter" for "a gee-pee-hyper..."?

**Comment by ashwinstar** [comment]
@danielgustafsson we (Jimmy and I) pushed this in to make sure its in before beta is cut as was approved and green.

**Comment by danielgustafsson** [comment]
> @danielgustafsson we (Jimmy and I) pushed this in to make sure its in before beta is cut as was approved and green.

Thanks!

**Pull request merged**